git-branch: replace --show-current with --contains example (#6387)
tldr-pages/tldr: pages/common/git-branch.md

`git branch -a`

> - Find what branch a Git commit came from:

Reviewer: ```suggestion
- Find which branch a Git commit came from:
```

Reviewer 2: Another suggetion:
```suggestion
- Show the branch a Git commit came from:
```

Author: ```suggestion
- List what branches includes a specific Git commit in their history:
```

Reviewer 3: A commit may be part of the history of multiple branches. What about this?

```suggestion
- List which branches include a specific Git commit in their history:
```
